### ISSUE TITLE
Problem in password plugin with dovecot encryption

### ISSUE THREAD

**Issue by pbraconnot** [roundcube/roundcubemail#7001]
Link of the file:
https://github.com/roundcube/roundcubemail/blob/master/plugins/password/password.php

fwrite function length is wrong as "\n" has 2 characters.
On line 722:

```
fwrite($pipes[0], $password . "\n", 1+strlen($password));
usleep(1000);
fwrite($pipes[0], $password . "\n", 1+strlen($password));
```
I think it should be

```
fwrite($pipes[0], $password . "\n", 2+strlen($password));
usleep(1000);
fwrite($pipes[0], $password . "\n", 2+strlen($password));
```

**Comment by alecpl** [comment]
No, "\n" is one character. Do you have any real issues with current code?

**Comment by pbraconnot** [comment]
> No, "\n" is one character. Do you have any real issues with current code?

Thank you @alecpl . Actually my problem was another. An include in dovecot.conf was causing doveadm pw output to be null and I got the message "doveconf: Fatal: Error in configuration file /etc/dovecot/dovecot.conf line 98: No matches".
Dovecot was working for login but not for saving passwords.

In /etc/dovecot/dovecot.conf I changed:

!include conf.d/*.conf
to
!include_try conf.d/*.conf

Save passwords is working now.
